.storybook/main.js

```diff
@@ -67,15 +67,15 @@ module.exports = {
         include: [modulesPath],
         exclude: /examples|stories|spec|codemod|docs/,
         loaders: [
```

Comment:
Will it work the same if change `loaders` to `use`? Webpack5 doesn't support `loaders` props.

Reply:
I'm guessing they mean the same thing, right?

Reply:
yes, they actually the same. I'm just not sure about the order